### DISCUSSION TITLE
Add a command to manually import python packages from anywhere in file

### DISCUSSION THREAD

**Discussion by Wakotu** [microsoft/pylance-release#3940] [Enhancement]
It's annoying for me to move to the beginning of the file and import packages. 
Although there is auto import function, it's not accurate enough. Sometimes I need a command to manually input the name of the package and it will import the package for me.
Just like this [C-mantic/add-include](https://bigbahss.github.io/vscode-cmantic/features/add-include/)

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
